config/services.yaml

```diff
@@ -25,6 +25,8 @@ parameters:
         anim_fle: Animateur.rice de cours de FLE
         psc1: PSC1
         nouveau: Nouveau bénévole
+    env(APP_NB_USERS): ~
```

Comment:
If we don't need this value for the run, can't we set them only in the `.env` file ?